packages/react-examples/src/react/ComboBox/ComboBox.FreeInput.Example.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { ComboBox, SelectableOptionMenuItemType } from '@fluentui/react';
+import type { IComboBoxOption, IComboBoxStyles } from '@fluentui/react';
+
+const options: IComboBoxOption[] = [
+  { key: 'Header1', text: 'First heading', itemType: SelectableOptionMenuItemType.Header },
+  { key: 'A', text: 'Option A' },
+  { key: 'B', text: 'Option B' },
+  { key: 'C', text: 'Option C' },
+  { key: 'D', text: 'Option D' },
+  { key: 'divider', text: '-', itemType: SelectableOptionMenuItemType.Divider },
+  { key: 'Header2', text: 'Second heading', itemType: SelectableOptionMenuItemType.Header },
+  { key: 'E', text: 'Option E' },
+  { key: 'F', text: 'Option F', disabled: true },
+  { key: 'G', text: 'Option G' },
+  { key: 'H', text: 'Option H' },
+  { key: 'I', text: 'Option I' },
+  { key: 'J', text: 'Option J' },
+];
```

Comment:
When all the options start with "Option" it makes trying out the auto-complete-freeinput behavior tricky. I'd recommend something that has more variation between options.